shared/translations/de_DE.json

```
@@ -0,0 +1,19 @@
+{
+    "Home": "Startseite",
+    "Recently updated": "Zuletzt geändert",
+    "Recently viewed": "Zuletzt gesehen",
+    "Created by me": "Von mir erstellt",
+    "Profile saved": "Profil gespeichert",
+    "Profile picture updated": "Profilfoto wurde aktualisiert.",
+    "Unable to upload new avatar": "Neuer Avatar kann nicht hochgeladen werden.",
```

While we're here this should change to profile picture to match the other language

```suggestion
    "Unable to upload new profile picture": "Neuer profilfoto kann nicht hochgeladen werden",
```

---

Left, the english version, is the string key which is in the source. Cannot change the text as you wish since its a key. gonna let it as it is